zcash_client_backend/proto/service.proto

```diff
@@ -174,7 +174,9 @@ service CompactTxStreamer {
     // Submit the given transaction to the Zcash network
     rpc SendTransaction(RawTransaction) returns (SendResponse) {}
 
-    // Return the txids corresponding to the given t-address within the given block range
+    // Return RawTransactions that match the given transparent address filter.
+    //
+    // Note: This function is misnamed, it returns complete `RawTransaction` values, not TxIds.
```

Comment:
Changes to `.proto` files must always be paired in their commits with the equivalent changes to the generated source code (CI checks for this).